ethereum_history_api/oracles/src/script/prepareJSFixtures.ts

```diff
@@ -9,8 +9,16 @@ import { writeObject } from '../util/file.js';
 import { RecordingClient, createRecordingClient } from '../ethereum/recordingClient.js';
 import { last } from '../util/array.js';
 
-async function createBlockFixture(client: RecordingClient, blockNumber: bigint): Promise<GetBlockFixture> {
-  await client.getBlock({ blockNumber });
+async function createBlockFixture(
+  client: RecordingClient,
+  blockNumber: bigint,
+  includeTransactions = false
+): Promise<GetBlockFixture> {
+  if (includeTransactions) {
+    await client.getBlock({ blockNumber, includeTransactions: includeTransactions });
+  } else {
+    await client.getBlock({ blockNumber });
+  }
   return last(client.getCalls()) as GetBlockFixture;
```

Comment:
Types should be different. This type has only tx hashes, but we should have a separate one which includes transactions